client/src/App.js

```diff
@@ -1,13 +1,13 @@
+/** @format */
```

Comment:
not sure why these are appearing at the top of the files, could you please remove

Reply:
I use prettier. when I save it auto came up.

Reply:
I use prettier. when I save it auto came up.